lib/client.js

```diff
@@ -615,6 +616,8 @@ class Parser {
       this.keepAlive += buf.toString()
     } else if (key.length === 14 && key.toString().toLowerCase() === 'content-length') {
       this.contentLength += buf.toString()
+    } else if (key.length === 16 && key.toString().toLowerCase() === 'transfer-encoding') {
```

Comment:
Judging by the line 617 (since I'm not very familiar with Undici's code), shouldn't this be `17`? `transfer-encoding` has 3 more characters than the previous header, and `.length` does indeed evaluate to `17`.

Reply:
Sounds like a test is missing for this?